resources/views/layouts/menu.blade.php

```diff
@@ -207,11 +207,21 @@ class="hidden-sm">@lang('Overview')</span></a>
                     @endif
                     @admin
                         <li role="presentation" class="divider"></li>
+                        <li class="dropdown-submenu"><a><i class="fa fa-braille fa-fw fa-lg"
+                                                        aria-hidden="true"></i> @lang('Manage Groups')</a>
+
+                            <ul class="dropdown-menu scrollable-menu">
                         @can('manage', \App\Models\DeviceGroup::class)
                             <li><a href="{{ url('device-groups') }}"><i class="fa fa-th fa-fw fa-lg"
-                                                                        aria-hidden="true"></i> @lang('Manage Groups')
+                                                                        aria-hidden="true"></i> @lang('Device Groups')
                                 </a></li>
                         @endcan
+                            <li><a href="{{ url('port-groups') }}"><i class="fa fa-link fa-fw fa-lg"
```

Comment:
I couldn't find this in the menu until I opened this file and looked for where it was.  Being under the device menu makes no sense.  It should be under the ports menu.

Reply:
main menu
-> device  -> device groups
-> port -> port groups
-> x -> x groups
-> y -> y groups
?

Reply:
yes

Reply:
changed